### PR TITLE
Provide all the key information in the provisioning service info command

### PR DESCRIPTION
**Pull request by cmickeyb**
Add some logging statements to get key information. Helps to debug problems with keys.

Signed-off-by: Mic Bowman <cmickeyb@gmail.com>

### CODE REVIEW

File: eservice/pdo/eservice/scripts/EServiceCLI.py [hunk 1]
```diff
@@ -96,6 +96,10 @@ def StartEnclaveService(config, enclave) :
         sys.exit(-1)
 
     logger.info('service started on port %s', http_port)
+    logger.warn('verifying_key: %s', enclave.verifying_key)
```
Comment (g2flyer) [inline eservice/pdo/eservice/scripts/EServiceCLI.py]: why are they warn?  Seems rather like info or even debug?

File: pservice/pdo/pservice/scripts/PServiceCLI.py [hunk 2]
```diff
@@ -78,6 +78,10 @@ def __init__(self, config, enclave) :
             'dataRequest' : self._datareq,
         }
 
+        logger.warn('start provisioning server')
+        logger.warn('verifying_key: %s', self.PSPK)
+        logger.warn('encryption_key: %s', self.EncryptionKey)
```
Comment (g2flyer) [inline pservice/pdo/pservice/scripts/PServiceCLI.py]: i'm surprised to find an encryption key for PS enclave. Is this there purely for symmetry reason to eservice enclaves?  Does anybody use that key? Given that ps enclave is "hidden" from outside and is more for plausible deniability of PS itself, having the PS encrypting any input to enclave doesn't really seem to make a difference in that argument? In any case, what seems sensitive is the enclave response but this should be encrypted the contract enclave encryption key which is presumably not this key referenced here?

Comment (cmickeyb) [reply pservice/pdo/pservice/scripts/PServiceCLI.py]: changed.

File: eservice/pdo/eservice/scripts/EServiceCLI.py (hunk 1 above)
Comment (cmickeyb) [reply eservice/pdo/eservice/scripts/EServiceCLI.py]: changed.

**Pull request merged**